packages/react/src/utils.ts

```diff
@@ -1,4 +1,4 @@
-export function lookupValue(value: Record<string, any>, ref: (string | number)[]) {
+export function lookupValue(value: unknown, ref: (string | number)[]) {
```

Comment:
fix: we accept any value and we check for different types inside (we also have unit tests for it)

Reply:
ok great, relying on inference here